### PR TITLE
Draft edit of proposed guidance content

### PR DESCRIPTION
**Pull request by calvin-lau-sig7**
Co-authored-by: frankieroberto <frankie@frankieroberto.com>

### CODE REVIEW

File: src/patterns/task-list-pages/index.md.njk [hunk 1]
```diff
@@ -77,29 +77,27 @@ Where possible, allow users to complete tasks in any order. This will help them
...
 
-![A screenshot showing a question that says 'Have you completed this section?'](have-you-completed-this-section.png)
-
-Add the question to your check your answers page when there are multiple tasks in a section or as the last question if the task is on a single page.
-
-If the user doesn’t select an option, display an error message:
+If the user selects ‘No, I’ll come back to it later,’ mark the task as “Incomplete” or “In progress”.
 
-![A screenshot showing a question that says 'Have you completed this section?'](have-you-completed-this-section-error.png)
+If the user selects ‘Yes, I have completed this section,’ mark the task as complete.
```
Comment (calvin-lau-sig7) [inline src/patterns/task-list-pages/index.md.njk]:
```suggestion
If the user selects ‘Yes, I've completed this section,’ mark the task as complete.
```

**Comment by frankieroberto** [comment]
@calvin-lau-sig7 shall I merge this, so that you can then continue committing to the fork in `x-govuk`?

**Comment by calvin-lau-sig7** [comment]
@frankieroberto Is there any way you can pull in a fork I'd made? 

@christopherthomasdesign has [created and inserted some examples in this commit](https://github.com/calvin-lau-sig7/govuk-design-system/commit/25689c0cca7a96357073c0330656d923046229fe#diff-91c168c712064172a92e4faad29a763c967593a52aa4f8638ac55960e83612f0). 

You can see what we've done as the most [recent 3 commits here](https://github.com/calvin-lau-sig7/govuk-design-system/commits/bbf12aeeca60ddd676c247f01b7e2f5a883a557a/src/patterns/task-list-pages/index.md.njk).

Otherwise, maybe just lift the code + guidance text? Apologies for the mess.

**Comment by frankieroberto** [comment]
> @frankieroberto Is there any way you can pull in a fork I'd made?

Done. You should be able to make any further commits direct to my branch. I’m on holiday now but will review in the new year! 🎄

**Pull request merged**